lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -26,11 +26,17 @@ def safe_add_enum_value(name, value)
   end
 
   def safe_add_column(table, column, type, options = {})
-    if options.has_key? :default
+    if options.has_key?(:default) && options[:default].include?("(")
       raise PgHaMigrations::UnsafeMigrationError.new(":default is NOT SAFE! Use safe_change_column_default afterwards then backfill the data to prevent locking the table")
```

Comment:
This message could include "with a dynamic value"

Reply:
@jcoleman @JonMR Can you approve this change?